### ISSUE TITLE
Eosinophil Expression

### ISSUE THREAD

**Issue by alead** [keyes-timothy/classes#5]
<img width="1017" alt="Screen Shot 2021-05-27 at 3 29 45 PM" src="https://user-images.githubusercontent.com/50126948/119905144-7d6af980-bf00-11eb-8b7f-f60319c80a25.png">

**Comment by alead** [comment]
I need to add in statistics, but there is statistical significance between SJIA-PAP and nonSJIA-PAP and SJIA-PAP and nonPAP

**Comment by keyes-timothy** [comment]
I'm not sure I understand the question.

**Comment by alead** [comment]
Lol no question, just wanted to show you what I found. I thought this would fit well with Aim 1 and hopefully the differential expression will confer this.

**Comment by keyes-timothy** [comment]
I didn't see differences in IFN-gamma expression between eosinophils, but I guess you did the statistics differently? I used the same mixed modeling (with FOVs as the unit of observation and patient_id's as a random effect) as the other analyses.

**Comment by alead** [comment]
So I ended up setting a threshold of positivity of a z-score of 2. Then took all the cells meeting that threshold and plotting their z-scored intensity as a violin plot.

**Comment by keyes-timothy** [comment]
Gotcha. Did you want to do that with all the markers? Instead of using means?

**Comment by alead** [comment]
Yeah that would be good to try out. In a supplemental figure, I am going to include a figure of the functional markers and their threshold of positivity.
![Figure_1](https://user-images.githubusercontent.com/50126948/119907249-b7d69580-bf04-11eb-8fcc-44b5309481b2.png)
Unfortunately, the intensity values are not bimodal, so I determined this treshold by looking to see at what point CD20 expression was present on nonBcells and set that as threshold

**Comment by keyes-timothy** [comment]
Alright I can experiment with that and see if I can make it work.

**Comment by alead** [comment]
Also, I would be hesistant with any results with PD1, so you can throw that one out too
